src/widgets/allergies/allergies-overview.test.tsx

```diff
@@ -39,12 +40,14 @@ describe("<AllergiesOverview />", () => {
     mockUseCurrentPatient.mockReturnValue([false, patient, patient.id, null]);
   });
 
+  const mockWidgetBasePath: widgetBasePath = { basePath: "/" };
+
   it("should display the patient's allergic reactions and their manifestations", async () => {
     mockPerformPatientAllergySearch.mockReturnValue(of(mockPatientAllergies));
 
     render(
       <BrowserRouter>
-        <AllergiesOverview basePath="/" />
+        <AllergiesOverview props={mockWidgetBasePath} />
```

Comment:
Yeah, something has gone terribly wrong. Let's fix the underlying problem, which is the mechanism by which props are passed to the widgets.

Reply:
I have pushed a fixed where I pass basePath as a prop, check out this PR [patient-chart](https://github.com/openmrs/openmrs-esm-patient-chart/pull/175).